vaccine_feed_ingest/runners/us/va_gov/normalize.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import os
+import pathlib
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+from vaccine_feed_ingest.utils.normalize import normalize_phone, normalize_url
+
+logger = getLogger(__file__)
+
+
+SOURCE_NAME = "us_giscorps_vaccine_providers"
+
+
+# def _get_availability(site: dict) -> schema.Availability:
```

Comment:
Can you remove unused code. If you want this code around then put it in a branch where we can run. Commented out code bitrots and it isn't clear when to re-enable it.

Reply:
yep! can do. this is mostly an artifact of the quick first pass mentality since this was a priority